css/full-width-pics.css

```diff
@@ -139,6 +139,103 @@
   font-family: sans-serif;
 }
 
+
+
+/****** Home Page Split Screen ********/
+
+.split{
+  display: flex;
+  height: 53vh;
+}
+
+.split-left{
+  width: 50%;
+  display: inline;
+  padding-top: 50px;
+  padding-right: 50px;
+  padding-left: 50px;
+  padding-bottom: 50px;
+  border-style: none solid none none;
```

Comment:
The solid line delineating between the left and right panes of content (i.e. `div.split-left` and `div.split-right`) do adhere to the wireframe mockup in #19, but it appears abrupt when actually implemented. I think that we should remove the borders for both panes, and allow the content to visually distinguish the different content areas. Please set `border-style: none` (or remove this property entirely).